developers/integrations/llm-frameworks/llamaindex/LlamaIndex.md

```diff
@@ -0,0 +1,44 @@
+
+2. [**Read and Listen**](#read-and-listen): Develop your conceptual understanding of these technologies.
+
+### Hands on Learning
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Data Loaders in LlamaIndex | Learn how to load data into Weaviate using LlamaIndex, and connect LlamaIndex to an existing Weaviate cluster. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/data-loaders-episode1/episode1.ipynb) |
+| Indexes in LlamaIndex | Learn about the various indexes you can build in LlamaIndex. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/indexes-episode2/indexes-in-llamaindex.ipynb) |
+| Recursive Query Engine | Learn how to build a recursive query engine. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/recursive-query-engine/recursive-retrieval.ipynb) |
+| Self Correcting Query Engine | Setup your vector store and build a self-correcting query engine. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/self-correcting-query-engine/self-correcting.ipynb) | 
+| Simple Query Engine | Build a simple query engine | [Notebook](https://github.com/weaviate/recipes/tree/main/integrations/llm-frameworks/llamaindex/simple-query-engine) |
+| SQL Router Query Engine | Build a SQL Query Engine to search through your vector and SQL database. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/sql-router-query-engine/sql-query-router.ipynb) |
+| Sub-Question Query Engine | Build a query engine that will break down a complex question into multiple parts. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/sub-question-query-engine/sub_question_query_enginev1.ipynb) |
+| Advanced RAG | This notebook walks your through an advanced Retrieval-Augmented Generation (RAG) pipeline using LlamaIndex and Weaviate. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/retrieval-augmented-generation/advanced_rag.ipynb) | 
+| Naive RAG | This notebook walks your through a naive Retrieval-Augmented Generation (RAG) pipeline using LlamaIndex and Weaviate. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/llm-frameworks/llamaindex/retrieval-augmented-generation/naive_rag.ipynb)
+
+
+
+### Read and Listen 
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Episode 1: Data Loading | This episode will show you how to connect your data to LlamaIndex and Weaviate.| [Video](https://youtu.be/Bu9skgCrJY8?feature=shared) | 
+| Indexes in LlamaIndex | We cover the three indexes (Vector Store Index, List Index, and Tree Index) and walk through the architecture design. The video ends with a demo using the Vector Store Index and List Index.  | [Video](https://youtu.be/6pLgOJrFL38?feature=shared) |
```

Comment:
Consider
The video covers the three LlamaIndex indexes (Vector Store Index, List Index, and Tree Index) and walks through the architecture design.